content/english/about/no-tech-for-ice.md

```diff
@@ -18,64 +18,43 @@ tags:
 
 As many of us are aware, technology plays an ever increasing role in our society and it is vital that our community has a voice in the decisions that are made.
 
-Techqueria’s mission is to advance Latinx professionals within the tech industry.
+Techqueria’s mission is to advance Latinx professionals within the tech industry. We want to take a moment to talk about how this impacts Techqueria as an organization.
 
-If we are successful in our mission, we will have people at the top with a seat at the table.
-
-As of now, there aren’t many of us within these companies making these decisions, but the decisions impact everyone from our communities back home.
-
-We want to take a moment to talk about how this impacts Techqueria as an organization.
-
-One of the ways we generate revenue as a nonprofit is by partnering with tech companies on events and digital marketing.
-
-This puts us in a situation where we must decide if it’s better for the community to work with companies that impact our members.
-
-Do we try to increase representation so these decisions are made more equitably or do we refuse a partnership based on a company’s track record?
-
-This is especially difficult because partnerships bring in resources that enables Techqueria to grow and advance our mission.
+One of the ways we generate revenue as a nonprofit is by partnering with tech companies on events and digital marketing. We must decide which companies to partner with and how those partnerships impact our community.
 
 This is where our values come in:
 
-1. Community First
-2. Radical Inclusion
-3. Over Communication
-4. Positive Vibes
-5. Data-Driven
+* Community First
+* Radical Inclusion
+* Over Communication
+* Positive Vibes
+* Data-Driven
 
 Our highest value is being community-first and in order for values to have meaning, we must apply them to actions.
 
-> **In particular, this translates Techqueria’s firm policy that  we will not partner with companies who are known to work with and support ICE.**
+> **In particular, this translates Techqueria’s firm policy that we will not partner with companies who are known to work with and support ICE.**
```

Comment:
There's some grammar error on my part - it should be

In particular, this translates to Techqueria having a firm policy that we will not partner with companies who are known to work with and support ICE.